orchestra/static/orchestra/timing/timecard/partials/timecard.html

```diff
@@ -2,107 +2,120 @@
+                  <em class="incomplete">Invalid Entries</em>
                 </span>
-                <span ng-hide="entry.description">
-                  <em>No description</em>
+                <span class="date-duration" ng-hide="vm.timeEntries.invalidEntriesForDate(dateEntries.$key).length">
+                  {{ vm.timeEntries.timeWorkedForDate(dateEntries.$key).humanizeUnits() }}
                 </span>
-              </div>
-              <div class="col-xs-4 edit-time" ng-if="!entry.editing">
-                {{ entry.time_worked.roundMinute().humanizeUnits() }}
-              </div>
-              <div class="col-xs-12" ng-if="!entry.editing">
-                <em class="pull-right">{{ vm.orchestraTasks.getDescription(vm.orchestraTasks.tasksByAssignmentId[entry.assignment]) || 'Unassigned' }}</em>
-              </div>
-
-              <!-- Editable view for description and time worked -->
-              <div class="col-xs-6" ng-if="entry.editing">
-                <input class="form-control"
-                       type="text"
-                       ng-model="entry.editData.description"
-                       placeholder="What did you work on?">
-              </div>
-              <div class="col-xs-4 edit-time" ng-if="entry.editing">
-                <div class="time-component">
-                  <input class="form-control"
-                         type="number"
-                         ng-model="entry.editData.timeWorked.h"
-                         min="0"
-                         max="23"
-                         enforce-integers>
-                  <label>hours</label>
-                </div>
-                <div class="time-component">
-                  <input class="form-control"
-                         type="number"
-                         ng-model="entry.editData.timeWorked.m"
-                         min="0"
-                         max="59"
-                         enforce-integers>
-                  <label>minutes</label>
+              </h2>
+              <div class="entry-row" ng-if="!dateEntries.length">
+                <div class="row">
+                  <div class="col-xs-12">
+                    <em>No entries for this date</em>
+                  </div>
                 </div>
               </div>
-            </div>
+              <div class="entry-row" ng-class="{'gray-stripe': $index % 2}" ng-repeat="entry in dateEntries | orderBy:'-id'">
+                <form class="container-fluid" novalidate name="entryForm" class="form-inline">
+                  <div class="row">
+                    <div class="col-xs-2 edit-tools disable-select" ng-if="!vm.expiredEntry(entry)">
+                      <i ng-click="vm.timeEntries.deleteEntry(entry)" class="fa fa-times-circle"></i>
+                      <i class="fa fa-pencil-square-o" ng-class="{active: entry.editing}" ng-click="vm.editEntry(entry)"></i>
```

Comment:
I don't think the button should mysteriously disappear, I think that would look weird. Going to leave as is.